emails/spec/mailers/spree/order_mailer_spec.rb

```diff
@@ -115,6 +119,7 @@
 
   context 'only shows eligible adjustments in emails' do
     before do
+      allow(Spree::BaseHelper).to receive(:locale_param)
```

Comment:
@MatthewKennedy this makes the test pass but the actual code will be broken :) 

we need to change this:
https://github.com/spree/spree/blob/ee6ffaa30b9469e4abe0eee60667baeaf0d3bbed/core/app/helpers/spree/base_helper.rb#L179

to:

```ruby
if defined?(locale_param) && locale_param.present?
```